src/main/java/com/tictactoe/Program.java

```diff
@@ -10,12 +10,22 @@
     private GameSettings menu = new GameSettings(receiver, validator);
     private GamePreference preference = menu.collectGamePreference();
```

Comment:
I am not sure what `GamePreference` is supposed to do here. Maybe consider a different name?